spark/dl/src/main/scala/com/intel/analytics/bigdl/models/inception/Train.scala

```diff
@@ -118,6 +120,19 @@ object TrainInceptionV1 {
       if (param.gradientL2NormThreshold.isDefined) {
         optimizer.setGradientClippingByl2Norm(param.gradientL2NormThreshold.get.toFloat)
       }
+      
+      if (param.visualization) {
+        val logdir = "bigdlSummary"
+        val timestamp: Long = System.currentTimeMillis / 1000
```

Comment:
can we convert the timestamp into a more human readable format? I have a similiar change in my local. I use the human readable time as the suffix. For your reference
```scala
val sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
val timeStamp = sdf.format(new Date())
```